src/simularium/versionHandlers.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable-next-line @typescript-eslint/no-var-requires */
+const si = require("si-prefix");
```

Comment:
minor red flag when you have to do this;  the library is old and not maintained.  Makes me want to look for another one, like unitsnet-js for example.  Alternative: write a small module wrapper file that "hides" the require statement.

Reply:
Yeah, it is an old library but it appealed to me because it's small and does exactly what we need for this specific purpose. But I agree we should look for a better one. We need to add a more comprehensive unit library for unit validation (like we talked about before) at some point, so maybe that'll be a good time to research and upgrade our unit library. I just looked into unitsnet-js but couldn't find in their docs how to use it to take 1.5e-9 m for example and turn it into 1.5 nm, but I might just need to look more carefully.

I did some Googling and found that I can actually just use `import * as si from "si-prefix";`, so changed it to that!